.env.default

```
@@ -38,7 +38,7 @@ PHP_IMAGE_VERSION=7.1
 
 # Set the version of the site audit script to use - you can use a tag or branch reference here
 # See https://github.com/govCMS/audit-site/releases
-SITE_AUDIT_VERSION=7.x-3.x-beta1
+SITE_AUDIT_VERSION=7.x-3.x-beta2
```

```suggestion
SITE_AUDIT_VERSION=7.x-3.x
```
Suggest removing version constraint whilst we're making changes to site-audit - ideally we have a local site process to check for updates automatically going forwards